lib/dor/datastreams/rights_metadata_ds.rb

```diff
@@ -59,11 +59,14 @@ def self.xml_template
       end.doc
     end
 
+    # NOTE: 8/6/2020 In the future, 'cdl' could be a valid option in the list below.  It will generate a <cdl> node
```

Comment:
Let's not build this until we know we'll need it.

Reply:
ok